main.js

```diff
@@ -20,17 +22,40 @@ saveBtn.addEventListener('click', (event) => {
 
 titleInput.addEventListener('keyup', activateSaveBtn)
 bodyInput.addEventListener('keyup', activateSaveBtn)
+showStarredIdeasBtn.addEventListener('click', createStarredIdeas)
+
 // star.addEventListener('click', changeFavoriteStatus)
```

Comment:
totally okay in your own repo, but sometimes leaving commented out code in production is no bueno, people won't merge your code in. instead you could make it a todo comment